mmif/utils/video_document_helper.py

```diff
@@ -127,6 +127,48 @@ def extract_mid_frame(mmif: Mmif, time_frame: Annotation, as_PIL: bool = False):
     return extract_frames_as_images(vd, [get_mid_framenum(mmif, time_frame)], as_PIL=as_PIL)[0]
 
 
+def get_representative_framenum(mmif: Mmif, time_frame: Annotation):
+    """
+    Calculates the representative frame number from an annotation.
+
+    :param mmif: :py:class:`~mmif.serialize.mmif.Mmif` instance
+    :param time_frame: :py:class:`~mmif.serialize.annotation.Annotation` instance that holds a time interval annotation containing a `representatives` property (``"@type": ".../TimeFrame/..."``)
+    :return: representative frame number as an integer
+    """
+    if 'representatives' not in time_frame.properties:
+        raise ValueError(f'The time frame {time_frame.id} does not have a representative.')
+    timeunit = time_frame.get_property('timeUnit')
+    video_document = mmif[time_frame.get_property('document')]
+    fps = get_framerate(video_document)
+    representatives = time_frame.get_property('representatives')
+    top_representative_id = representatives[0]
+    views = mmif.get_all_views_contain(AnnotationTypes.TimePoint)
+    representative_timepoint_anno = None
+    for view in views:
+        try:
+            representative_timepoint_anno = view.get_annotation_by_id(top_representative_id)
+            break
+        except KeyError:
+            continue
```

Comment:
this can be much shorter if using 
https://github.com/clamsproject/mmif-python/blob/2a5bffc36ba360e45715bd48c7af07e841717f7a/mmif/serialize/mmif.py#L564-L575 
and 
https://github.com/clamsproject/mmif-python/blob/693972a2868e226453f55b9c42380061ce96d996/mmif/serialize/annotation.py#L130-L135